get_intervals.py

```diff
@@ -174,8 +182,12 @@ def main():
                 len(train_peaks), len(train_nonpeaks)))
 
         # Write to file
-        train.to_csv(args.prefix + '.training_intervals.bed',
-                     sep='\t', index=False, header=False)
+        if args.prefix is None:
+            out_file_name = 'training_intervals.bed'
+        else:
+            out_file_name = args.prefix + '.training_intervals.bed'
+        out_file_path = args.out_home + '/' + out_file_name
```

Comment:
`os.path.join` to concatenate file paths

Reply:
done